src/scripts/main.js

```diff
@@ -1,3 +1,391 @@
+optionsTokyo.setAttribute('value', 'Tokyo');
+optionsSingapore.textContent = 'Singapore';
+optionsSingapore.setAttribute('value', 'Singapore');
+
+optionsLondon.textContent = 'London';
+optionsLondon.setAttribute('value', 'London');
+
+optionsNewYork.textContent = 'New York';
+optionsNewYork.setAttribute('value', 'New York');
+
+optionsEdinburgh.textContent = 'Edinburgh';
+optionsEdinburgh.setAttribute('value', 'Edinburgh');
+
+optionsSanFrancisco.textContent = 'San Francisco';
+optionsSanFrancisco.setAttribute('value', 'San Francisco');
+
+selectOffice.append(
+  optionsTokyo,
+  optionsSingapore,
+  optionsLondon,
+  optionsNewYork,
+  optionsEdinburgh,
+  optionsSanFrancisco,
+);
+
+labelOffice.append(selectOffice);
+selectOffice.required = true;
+
+inputAge.setAttribute('data-qa', 'age');
+inputAge.setAttribute('name', 'age');
+
+inputAge.setAttribute('type', 'number');
+labelAge.appendChild(inputAge);
+inputAge.required = true;
+
+inputSalary.setAttribute('data-qa', 'salary');
+inputSalary.setAttribute('name', 'salary');
+inputSalary.setAttribute('type', 'number');
+inputSalary.required = true;
+labelSalary.appendChild(inputSalary);
+
+buttonSubmit.textContent = 'Save to table';
+buttonSubmit.setAttribute('type', 'submit');
+form.appendChild(labelName);
+form.appendChild(labelPosition);
+form.appendChild(labelOffice);
+form.appendChild(labelAge);
+form.appendChild(labelSalary);
+form.appendChild(buttonSubmit);
+form.setAttribute('type', 'submit');
+
+document.body.append(form);
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const newRows = document.createElement('tr');
+  const newName = document.createElement('td');
+  const newPosition = document.createElement('td');
+  const newOffice = document.createElement('td');
+  const newAge = document.createElement('td');
+  const newSalary = document.createElement('td');
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.setAttribute('class', 'notification');
+
+  div.style.position = 'absolute';
+
+  const valid = checkValidValueError();
+
+  if (valid) {
+    createMessageError(div);
+  } else {
+    createMessagSaccess(div);
+  }
+
+  document.body.append(div);
+
+  if (div.getAttribute('data-qa') === 'error') {
+    div.style.background = 'rgb(153, 41, 26)';
+
+    setTimeout(() => {
+      div.style.display = 'none';
+    }, 4000);
+  }
+
+  newName.textContent = inputName.value;
+  newPosition.textContent = inputPosition.value;
+  newOffice.textContent = selectOffice.value;
+  newAge.textContent = inputAge.value;
+  newSalary.textContent = `$${(+inputSalary.value).toLocaleString('en-US')}`;
+
+  newRows.append(newName, newPosition, newOffice, newAge, newSalary);
+
+  setTimeout(() => {
+    div.style.display = 'none';
+  }, 4000);
+
+  const validSaccess = checkValidValueSaccess();
+
+  if (validSaccess) {
+    tbody.append(newRows);
+    form.reset();
+  }
+});
+
+title.addEventListener('click', (e) => {
```

Comment:
The variable 'sortTody' seems to be a typo. Consider renaming it to 'sortedBody' for clarity.